ngi_reports/common/ign_sample_report.py

```diff
@@ -48,8 +49,9 @@ def __init__(self, config, LOG, working_dir, **kwargs):
         self.parse_picard_metrics()
 
         # Plot graphs
-        self.LOG.info('Plotting graphs')
-        self.make_plots()
+        if create_plots:
```

Comment:
Ok, fair enough - I think it's probably simpler to keep it always making plots and `ctrl-c` if they're not needed? (see below).

I'm trying to resist making too many command line options basically, as I think a broad package like this could end up with hundreds if we're not careful.